Consul.Test/ConfigurationTest.cs

```diff
@@ -49,5 +49,66 @@ public async Task Configuration_ApplyConfig(string protocol)
             Assert.Equal(payload.Kind, queryResult.Response.Kind);
             Assert.Equal(payload.Protocol, queryResult.Response.Protocol);
         }
+
+        [Fact]
+        public async Task Configuration_ListConfig()
+        {
+            var firstPayload = new ServiceDefaultsEntry
+            {
+                Kind = "service-defaults",
+                Name = "web",
+                Protocol = "https"
+            };
+
+            var secondPayload = new ServiceDefaultsEntry
+            {
+                Kind = "service-defaults",
+                Name = "db",
+                Protocol = "https"
+            };
+            var writeResult = await _client.Configuration.ApplyConfig(firstPayload);
+            Assert.Equal(HttpStatusCode.OK, writeResult.StatusCode);
+            writeResult = await _client.Configuration.ApplyConfig(secondPayload);
+            Assert.Equal(HttpStatusCode.OK, writeResult.StatusCode);
+            var queryResult = await _client.Configuration.ListConfig<ServiceDefaultsEntry>(firstPayload.Kind);
+            var configurations = queryResult.Response;
+            var webConfig = configurations.SingleOrDefault(c => c.Name == firstPayload.Name);
+            var dbConfig = configurations.SingleOrDefault(c => c.Name == secondPayload.Name);
+
+            Assert.Equal(firstPayload.Name, webConfig.Name);
+            Assert.Equal(firstPayload.Kind, webConfig.Kind);
+            Assert.Equal(firstPayload.Protocol, webConfig.Protocol);
```

Comment:
`webConfig` and `dbConfig` could be null (theoretically), so ideally it would be to have two more asserts:
```
Assert.NotNull(dbConfig);
Assert.NotNull(webConfig);
```

Reply:
that's true. thanks